CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 * Support for adding description in Search Configuration ([#293](https://github.com/opensearch-project/search-relevance/pull/293))
 
 ### Bug Fixes
+* Removed deprecated `AccessController.doPrivileged()` usage in `JsonUtils` to prevent warnings and ensure compatibility with newer Java versions. ([#307](https://github.com/opensearch-project/search-relevance/pull/307))
```

Comment:
this is more a Maintenance change, correct me if I'm wrong but that warning is not in any user-facing elements (exception or UI control)